### DISCUSSION TITLE
IGV and bam files

### DISCUSSION THREAD

**Discussion by gsk7757** [biostars/biostar-handbook-issues#319] [Q&A]
Hello, 
I have performed the RNA-seq analysis on a bacteria. After the analysis, I tried to visualize the bam files by using IGV. However, I encountered an issue where, for a particular gene, no reads were mapped in IGV. Interestingly, when I used the same bam file to determine the counts using featurecounts, I observed counts for that gene. What could be causing this discrepancy?

**Comment by ialbert** [comment]
Sometimes you are not zoomed in enough or the track is not autoscaling, and that causes misdirection

Check that the track is on autoscale first and that you are zoomed in.

Also check that the annotation are the same, could be a different build or something.

In general, it would not be possible to have featurecounts report a count, but IGV does not show something.

The other way  around it could happen . Feature counts could show less than IGV (because feature counts can be more restrictive in what it counts).

**Reply by gsk7757** [reply]
Thank you so much for such a detailed response. Yes, I have checked the autoscale before zooming in and yes, my featurecounts report doesn't show any counts for 7 genes but I see some reads mapping in the IGV. However, there is one gene where I see no read-mapping to the entire length of the gene (even after zooming in to the maximum), but I have counts for that gene. I am using the same annotation file for feature counts as IGV.
